### ISSUE TITLE
Argument "filesToInclude" is invalid, use "files-to-include"

### ISSUE THREAD

**Issue by harshgour** [Reggionick/s3-deploy#112]
@Reggionick 

The input code mentioned in the action script is -> core.getInput('files-to-include')

<img width="964" alt="image" src="https://user-images.githubusercontent.com/60436404/198959024-e0d395b1-a163-415f-b9de-42ba5dccec61.png">

Thus, the input defined in actions.yml should be 'files-to-include' instead of 'filesToInclude' because it is throwing warning while deploying. And also in all the places for documentation purposes.

P.S. - Also, update the node version.

**Comment by cornwe19** [comment]
It's also worth noting on this issue that using comma separated values for the patterns supplied to `files-to-include` doesn't appear to work as documented. I was only able to get this working by space-separating multiple patterns

For example, `files-to-include: ".well-known/*,./**/*"` yields deployment output `Deploying files: []` whereas, `files-to-include: ".well-known/* ./**/*"` yields deployment output like
```
Deploying files: [
  './.well-known/apple-app-site-association',
  './.well-known/apple-developer-merchantid-domain-association',
  './500.html',
  ...
```

**Comment by Reggionick** [comment]
Fixed with #89
